File: app/src/main/java/com/padmitriy/android/currcalc/network/RevoApi.java
package com.padmitriy.android.currcalc.network;

import com.padmitriy.android.currcalc.model.RatesResponse;
import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface RevoApi {

    @GET("latest")
    Observable<RatesResponse> getRates(@Query("base") String baseCurrency);
}

File: app/src/main/java/com/padmitriy/android/currcalc/network/RevoApi.kt
package com.padmitriy.android.currcalc.network

import com.padmitriy.android.currcalc.model.RatesResponse
import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Query

interface RevoApi {

    @GET("latest")
    fun getRates(@Query("base") baseCurrency: String): Single<RatesResponse>
}
